doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+
+The following figure illustrates the background mesh as well as the particles used to represent the IB on top of it:
+
+.. image:: images/taylor_couette_with_nitsche_mesh.png
+    :alt: The background mesh and the embedded particles
+    :align: center
+    :name: nitsche_particles
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The ``boundary conditions`` subsection becomes simple since the inner cylinder boundaries were specified in the previous section.
+
+.. code-block:: text
+
+  subsection boundary conditions
+    set number                  = 1
+      subsection bc 0
+          set type              = noslip
+      end
+  end
+
+
+First, the ``number`` of boundary conditions to be applied must be specified. For each boundary condition, the ``id`` of the boundary as well as its ``type`` must be specified. The outer cylinder (``0``) is static and, consequently, a ``noslip`` boundary condition is applied. 
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~
+
+The analytical solution for the Taylor-Couette problem is only valid at low Reynolds number. We thus set the kinematic viscosity to 1.
+
+.. code-block:: text
+
+  subsection physical properties
+    subsection fluid 0
+      set kinematic viscosity            = 1.0
+    end
+  end
+
+
+FEM interpolation
+~~~~~~~~~~~~~~~~~
+
+Lethe supports the use of arbitrary interpolation order. The :math:`\mathcal{L}^2` norm of the error is :math:`\mathcal{O}\left(h^{n+1} \right)` where :math:`h` is a measure of the element size and `n=1` is the interpolation order of the velocity. However, since the torque applied on the inner cylinder depends on the deviatoric stress tensor, which depends on the velocity gradient, it's error will be :math:`\mathcal{O}(n)`. Taking this into account, we use second order polynomials in this example to obtain higher accuracy on the torque. We specify the interpolation order for both pressure and velocity using the ``FEM`` subsection:
+
+.. code-block:: text
+
+    subsection FEM
+        set velocity order            = 1
+        set pressure order            = 1
+    end
+
+.. warning:: 
+    With the ``gls_navier_stokes_2d`` solver, Q2-Q2 elements could also be used. However, we have not found that these lead to better results when the flows are at a low Reynolds number.
+
+
+Analytical solution
+--------------------
```

Comment:
Not the good heading I think!
You are still explaining the Parameter file section of your doc, so it needs tilde symbols.
But that is my opinion :)

Reply:
Oh! The warning was already there just to specify that you can also use this in the FEM section. Maybe this should be moved to the Features section?

Reply:
As you wish, maybe it is better in the Features because you don't specify anything in the prm file in this section.

Reply:
Ok I did your suggestion :)